network/manifest-1/POST_GENESIS.md

```diff
@@ -1,28 +1,101 @@
 # Post-Genesis
 
-### Become a validator
-* Install the manifest binary
-* manifestd init <moniker> --chain-id manifest-1 --default-denom poastake
-* Replace your genesis with the public one found in this repo
-* Find peers and seeds here.
-* Update your minimum-gas-prices in the app.toml
-* Start the node and sync up
-* Once completed, `manifestd tx poa create-validator path/to/validator.json --from keyname`. This command shows the JSON needed.
-```json
+## Become a validator
+
+### Install the manifest binary
+
+```bash
+# Clone git repository
+git clone https://github.com/liftedinit/manifest-ledger.git
+cd manifest-ledger
+git checkout VERSION
+
+make install # go install ./...
+# For ledger support `go install -tags ledger ./...`
+manifestd config set client chain-id manifest-1
+```
+
+OR
+
+```bash
+wget <link to manifest precompile>
```

Comment:
We should add the binary release to the CI.

Reply:
Added for the following systems (pretty much all) on tag

3c9da62396105086f3b70bb90f7e564455b33e93

![image](https://github.com/liftedinit/manifest-ledger/assets/31943163/f5cbceb4-eafe-4bfd-8224-01daded10bad)